plugins/aea-ledger-ethereum/aea_ledger_ethereum/ethereum.py

```diff
@@ -1436,8 +1436,16 @@ def build_transaction(  # pylint: disable=too-many-arguments
         :param raise_on_try: whether the method will raise or log on error
         :return: the transaction
         """
+
+        if method_args is None:
```

Comment:
These are always kwargs not args, let's rename

Reply:
That'd be a breaking change now. I suggest opening an issue instead.